[2022-09-26T16:32:45.147Z] error: Failed to load services.yaml, please check for errors
[2022-09-26T16:32:45.150Z] error: bad indentation of a sequence entry (23:6)

 20 |         description: Postgres Instance
 21 |         server: my-local-docker # Th ...
 22 |         container: database-dev # Th ...
 23 |      - Surreeal:
-----------^
 24 |         icon: postgres.png
 25 |         href: "https://database.scho ...
[2022-09-26T16:32:45.795Z] error: Failed to load services.yaml, please check for errors
[2022-09-26T16:32:45.796Z] error: bad indentation of a sequence entry (23:6)

 20 |         description: Postgres Instance
 21 |         server: my-local-docker # Th ...
 22 |         container: database-dev # Th ...
 23 |      - Surreeal:
-----------^
 24 |         icon: postgres.png
 25 |         href: "https://database.scho ...
[2022-09-26T16:32:57.621Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.host.or.ip
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.host.or.ip'
  }
]
[2022-09-26T16:33:02.942Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.host.or.ip
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.host.or.ip'
  }
]
[2022-09-26T16:33:09.765Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.host.or.ip
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.host.or.ip'
  }
]
[2022-09-26T16:33:16.038Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.host.or.ip
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.host.or.ip'
  }
]
[2022-09-26T16:33:18.853Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.host.or.ip
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.host.or.ip'
  }
]
[2022-09-26T16:36:54.683Z] error: undefined
[2022-09-26T16:36:59.895Z] error: undefined
[2022-09-26T16:37:05.758Z] error: undefined
[2022-09-26T16:37:06.772Z] error: undefined
[2022-09-26T16:37:11.912Z] error: undefined
[2022-09-26T16:37:27.160Z] error: undefined
[2022-09-26T16:37:46.822Z] error: undefined
[2022-09-26T17:53:25.735Z] error: end of the stream or a document separator is expected (15:1)

 12 |     focus: true # Optional, wil ...
 13 |     target: _blank # One of _se ...
 14 | 
 15 | format:
------^
 16 |   dateStyle: short
 17 |   timeStyle: short
[2022-09-26T17:53:26.024Z] error: end of the stream or a document separator is expected (15:1)

 12 |     focus: true # Optional, wil ...
 13 |     target: _blank # One of _se ...
 14 | 
 15 | format:
------^
 16 |   dateStyle: short
 17 |   timeStyle: short
[2022-09-26T17:53:29.262Z] error: end of the stream or a document separator is expected (15:1)

 12 |     focus: true # Optional, wil ...
 13 |     target: _blank # One of _se ...
 14 | 
 15 | format:
------^
 16 |   dateStyle: short
 17 |   timeStyle: short
[2022-09-26T17:53:39.457Z] error: end of the stream or a document separator is expected (15:1)

 12 |     focus: true # Optional, wil ...
 13 |     target: _blank # One of _se ...
 14 | 
 15 | format:
------^
 16 |   dateStyle: short
 17 |   timeStyle: short
[2022-09-26T22:12:58.404Z] error: undefined
[2022-09-26T22:12:58.442Z] error: Failed to load services.yaml, please check for errors
[2022-09-26T22:12:58.443Z] error: undefined
[2022-09-27T04:10:59.613Z] error: undefined
[2022-09-28T07:29:10.963Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:13.363Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:18.462Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:21.068Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:33.579Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:46.193Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:29:59.420Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:05.721Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:10.322Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:10.831Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:11.345Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:12.434Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:18.855Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:22.994Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T07:30:23.991Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T14:02:41.437Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T14:02:46.545Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-28T14:02:56.678Z] error: [
  500,
  Error: getaddrinfo ENOTFOUND traefik.schoolbot.devother
      at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:111:26) {
    errno: -3008,
    code: 'ENOTFOUND',
    syscall: 'getaddrinfo',
    hostname: 'traefik.schoolbot.devother'
  }
]
[2022-09-29T23:21:29.940Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:21:40.054Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:21:55.205Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:08.024Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:13.143Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:38.301Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:45.316Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:46.484Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:51.617Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:22:56.003Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:01.124Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:01.767Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:16.275Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:21.921Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:23.429Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:41.478Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:23:46.608Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:01.454Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:10.695Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:11.720Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:15.439Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:20.833Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
[2022-09-29T23:24:23.549Z] error: [
  500,
  Error: self-signed certificate
      at TLSSocket.onConnectSecure (node:_tls_wrap:1538:34)
      at TLSSocket.emit (node:events:513:28)
      at TLSSocket._finishInit (node:_tls_wrap:952:8)
      at ssl.onhandshakedone (node:_tls_wrap:733:12) {
    code: 'DEPTH_ZERO_SELF_SIGNED_CERT'
  }
]
